### PR TITLE
whitespace and fixed test mode bug

### PR DESCRIPTION
**Pull request by sshah-1**
- BUG: sword beam will be used when using sword if the b item is the magical rod, regardless of link's health
- tried to make classes smaller via removing white space, and combining some fields 
(eg. instead of: private int a; private int b; changed to: private int a,b;)
- changed color magic strings for gohma into color
- fixed test-mode bug
- (hopefully) fixed rings being op
- deleted a bunch of stuff imn hudinventory (to be replaced)

### CODE REVIEW

File: sprint0/sprint0/Enemies/Snake.cs [hunk 1]
```diff
@@ -31,49 +31,39 @@ public Snake(Texture2D texture, Vector2 location, Game1 game) : base(texture, lo
...
-                    {
-                        direction = Direction.West;
-                    }
-                }
-                
+            currentFrame = (currentFrame + 1) % (totalFrames * repeatedFrames);
+            if (direction == Direction.West)
+            {
+                spriteEffect = SpriteEffects.FlipHorizontally;
+                Location = new Rectangle(Location.X - 1, Location.Y, Location.Width, Location.Height);
```
Comment (SnobbyDragon) [inline sprint0/sprint0/Enemies/Snake.cs:52]: ok I feel like we should make a shift rectangle method or something bc we can already easily convert direction to a vector 2 using our extension method, but all these cases are necessary for rectangle :(

**Review by SnobbyDragon — APPROVED** [review]

File: sprint0/sprint0/Enemies/Snake.cs (hunk 1 above)
Comment (sshah-1) [reply sprint0/sprint0/Enemies/Snake.cs:52]: yea... to be honest I wasn't looking too closely at the functions of everything though

**Review by SnobbyDragon — DISMISSED** [review]

**Review by Hajohnson3m — APPROVED** [review]

File: sprint0/sprint0/Bosses/Gleeok/GleeokNeck.cs [hunk 2]
```diff
@@ -8,14 +8,14 @@ namespace sprint0
...
+        private readonly Game1 game;
         public Rectangle Location { get; set; }
         public Texture2D Texture { get; set; }
-        private List<IEnemy> neck;
+        private readonly List<IEnemy> neck;
         public int Damage { get => 0; }
         private bool isDead;
         public EnemyType Type { get => EnemyType.Gleeok; }
-        IEnemy body;
+        readonly IEnemy body;
```
Comment (SnobbyDragon) [inline sprint0/sprint0/Bosses/Gleeok/GleeokNeck.cs:18]: im not sure where this is accessed but can we make it private if possible

Comment (SnobbyDragon) [reply sprint0/sprint0/Bosses/Gleeok/GleeokNeck.cs:18]: or into a public property if needed

**Review by SnobbyDragon — APPROVED** [review]
approved w/minor comment

**Pull request merged**